src/qt/bitcoin.cpp

```diff
@@ -29,12 +29,12 @@
 
 #include <interfaces/handler.h>
 #include <interfaces/node.h>
+#include <node/ui_interface.h>
```

Comment:
It seems `qt/bitcoin.cpp` does not use any symbols from `node/ui_interface.h` directly.

Reply:
thx, removed